src/store/ProductsStore/ProductsStore.ts

```diff
@@ -0,0 +1,139 @@
+import axios from 'axios';
+import { action, computed, IReactionDisposer, makeObservable, observable, reaction, runInAction } from 'mobx';
+
+import { Option } from '@components/MultiDropDown/MultiDropDown';
+import { getApiUrl } from '@config/api';
+import { Meta } from '@config/meta';
+import { ProductModel } from '@store/models/Products/ProductModel';
+import {
+  CollectionModel,
+  getInitialCollectionModel,
+  linearizeCollection,
+  normalizeCollection,
+} from '@store/models/shared/collectionModel';
+import rootStore from '@store/RootStore';
+
+import { normalizeRawProduct } from '../models/Products';
```

Comment:
Можно добавить  `export * from './ProductModel';` в файл src/store/models/Products/index.ts и после этого объединить тут импорты:

```suggestion
import { normalizeRawProduct, ProductModel } from '@store/models/Products';
```